transmart-copy/src/main/groovy/org/transmartproject/copy/table/TreeNodes.groovy

```diff
@@ -152,4 +153,32 @@ class TreeNodes {
         }
     }
 
+    private updateTreeNodePaths(Map<String, String> oldToNewConceptPath) {
```

Comment:
Aren't we now again hardcoding conventions about concept path and tree path to be the same?